contrib/influxdb/fluo_metrics_setup.txt

```diff
@@ -13,7 +13,7 @@
 
 # DDL
 CREATE USER fluo WITH PASSWORD 'secret' WITH ALL PRIVILEGES
-CREATE DATABASE IF NOT EXISTS fluo_metrics
+CREATE DATABASE fluo_metrics
```

Comment:
What's the behavior if it exists now?

Reply:
If it exists, the database won't get created and manually running the `influx -import` on the metrics host throws the below error. Also, this doesn't fail the muchos setup because the import command still produces a return code of '0' despite the error. 
```bash
error: error parsing query: found NOT, expected ; at line 1, char 20
error: database not found: fluo_metrics
```

Reply:
sorry, I mean, after your change, what happens in this script if the database already exists? do the other lines work, or is the rest aborted?

Reply:
If the database already exists, the rest of the lines still work. To prove, I did a quick test by manually creating the `fluo_metrics` database using the CLI, run the import script & could see it works.

Reply:
Cool. Thanks!